bin/sass-lint.js

```diff
@@ -10,6 +10,12 @@ var configPath,
     configOptions = {},
     exitCode = 0;
 
+var tooManyWarnings = function (detects) {
+  var warningCount = lint.warningCount(detects).count;
+
+  return warningCount > 0 && warningCount > program.maxWarnings;
+};
+
 var detectPattern = function (pattern) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/snugug/sass-lint/file/2359663576/issues/source?bid=3169912&fileBranchId=3171901#l19)